diff --git a/scribus.appdata.xml.in b/scribus.appdata.xml.in
index c38ef6b..e753dc9 100644
--- a/scribus.appdata.xml.in
+++ b/scribus.appdata.xml.in
@@ -1,12 +1,14 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!-- Copyright 2016 Peter Linnell  plinnell@scribus.net, The Scribus Team 2002-2025> -->
-<component>
+<component type="desktop-application">
   <id type="desktop">scribus@TAG_VERSION@.desktop</id>
   <metadata_license>CC0-1.0</metadata_license>
   <name>Scribus</name>
   <project_license>GPL-2.0+</project_license>
   <summary>Open Source Page Layout and Desktop Publishing (DTP)</summary>
   <url type="homepage">https://www.scribus.net/</url>
+  <url type="bugtracker">https://bugs.scribus.net/</url>
+  <url type="vcs-browser">https://wiki.scribus.net/canvas/Checking_out_SVN</url>
   <description>
   <p>Scribus is a cross-platform open source page layout program which
 produces commercial grade output in PDF and Postscript.</p>
@@ -31,7 +33,9 @@
     </screenshot>
   </screenshots>
   <update_contact>scribus-dev@lists.scribus.net</update_contact>
-  <content_rating type="oars-1.0"/>
+  <content_rating type="oars-1.1" />
+  <developer_name>Scribus developers</developer_name>
+  <launchable type="desktop-id">net.scribus.Scribus.desktop</launchable>
   <releases>
     <release version="1.6.3" date="2025-01-08" type="stable"/>
     <release version="1.6.2" date="2024-06-15" type="stable"/>
